core/src/epicli/data/common/ansible/playbooks/roles/repository/tasks/setup.yml

```diff
@@ -76,3 +76,7 @@
 
 - name: Set up repositories
   include_tasks: "{{ ansible_os_family }}/setup.yml"
+
+- name: Include Helm repository creation
+  include_tasks: "create-helm-repo.yml"
+  when: inventory_hostname in target_repository_hostnames[0]
```

Comment:
This is a "string" match, because `target_repository_hostnames[0]` is a string (not a list), is this intended? I think using `==` here (a strict match) would be the correct approach. :+1: